### PR TITLE
CDPS-135 offences page - new design revisions

### PR DESCRIPTION
**Pull request by jamesgrant-moj**

### CODE REVIEW

File: server/views/partials/offencesPage/courtCaseContent.njk [hunk 1]
```diff
@@ -1,97 +1,170 @@
...
-                            </dd>
-                        </div>
-                        <div class="govuk-summary-list__row">
-                            <dt class="govuk-summary-list__key">
-                            {{ sentence.summaryDetailRows[1].label }}
-                            </dt>
-                            <dd class="govuk-summary-list__value">
-                            {{ sentence.summaryDetailRows[1].value }}
-                            </dd>
+        {{ sentence.courtDateResultsPrimaryCode | dump }}
```
Comment (davidmiddletonmoj) [inline server/views/partials/offencesPage/courtCaseContent.njk:53]: Should this be here?

**Review by davidmiddletonmoj — COMMENTED** [review]
Just a few comments to look into

**Comment by jamesgrant-moj** [comment]
Hi @davidmiddletonmoj, thanks for taking a look. I have updated the PR based on your feedback.

**Review by davidmiddletonmoj — APPROVED** [review]
LGTM

**Pull request merged**